src/main/java/org/jenkinsci/plugins/gitclient/Git.java

```diff
@@ -136,6 +141,29 @@ public GitClient invoke(File f, VirtualChannel channel) throws IOException, Inte
     }
 
     /**
+     * Retrieve the {@link Launcher} to be used to run git cli.
+     */
+    private @CheckForNull Launcher getLauncher() {
```

Comment:
@jglick is there any way to retrieve the Node's Launcher in a workflow context ?